CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+# dbt_stripe_source v0.14.1
+[PR #91](https://github.com/fivetran/dbt_stripe_source/pull/91) includes the following updates:
+
+## Bug Fixes
+- Fixed a Snowflake error in the `does_table_exist` macro that could occur when another package using quoted table or column names runs concurrently.
+- Updated the following models to dynamically select the correct source based on available tables:
+  - `stg_stripe__price_plan_tmp`
+  - `stg_stripe__subscription_tmp`
+
+## Under the Hood
+- Improved the `does_table_exist` macro:
+  - Added a `dispatch` to prevent conflicts with macros of the same name in other packages.
+  - Now returns `'exists'` only when the table is present, instead of returning `'True'` or `'False'`. This avoids issues where Jinja interpreted those strings as truthy values rather than actual booleans.
```

Comment:
I would add another bit about how `'exists'` is then used downstream
```suggestion
  - Now returns `'exists'` only when the table is present, instead of returning `'True'` or `'False'`. This avoids issues where Jinja interpreted both those strings as truthy values rather than actual booleans. The `'exists'` output is then used as an exact match to determine variable configurations depending on the presence of tables. 
```

Something like that?